app/models/candidate_interface/reference/referee_type_form.rb

```diff
@@ -0,0 +1,19 @@
+module CandidateInterface
+  class Reference::RefereeTypeForm
+    include ActiveModel::Model
+
+    attr_accessor :referee_type
+
+    validates :referee_type, presence: true
```

Comment:
How about a `validates :referee_type, inclusion: { in: REFEREE_TYPES }`?

Reply:
Do you mean using a constant `REFEREE_TYPES` to store the possible types? I've added an enum to the references model in the next PR I'm working on:
```
enum referee_type: {
    academic: 'Academic',
    professional: 'Professional',
    school_based: 'School-based',
    character: 'Character',
  }
```
So it doesn't allow any other values as a referee_type, just like we did with status. What do you think?

Reply:
I'll wait for your comment before merging this @tvararu  :)

Reply:
Whichever way works! You can also add it in the next PR 👍 You can re-use an enum or just an array of strings